components/schema-migrator/main.go

```diff
@@ -66,6 +66,11 @@ func invokeMigration() error {
 	_, present := os.LookupEnv("DB_SSL")
 	if present {
 		dbName = fmt.Sprintf("%s?sslmode=%s", dbName, os.Getenv("DB_SSL"))
+
+		_, present := os.LookupEnv("DB_SSLROOTCERT")
+		if present {
+			dbName = fmt.Sprintf("%s&sslrootcert=%s", dbName, os.Getenv("DB_SSLROOTCERT"))
```

Comment:
is it ok to log SSL certificate?

Reply:
hmm It's formatting connection string, not log as far as I see. On the other hand, because it's a CA cert, it's just used for encrypted traffic. (not used for authentication etc)

Reply:
sorry, you are righht